pkg/vulnsrc/cargo/cargo.go

```diff
@@ -22,6 +24,10 @@ const (
 	cargoDir = "rust-advisory-db"
 )
 
+// https://github.com/RustSec/advisory-db/issues/414#issuecomment-702197689
+var regexAdvisoryMarkdown = regexp.MustCompile(
+	`(?sm)^\x60\x60\x60toml\s*?$(.*?)^\x60\x60\x60\s*?$\s*^#\s*([^\r\n]+)\s*?$(.*)`)
```

Comment:
I feel `regex` is difficult to maintain. Could you use `scanner.Scan` and `scanner.Text`?
https://golang.org/pkg/bufio/#Scanner

I'm sorry to bother you again. If you are busy, I'll work on it since we have to fix this issue as soon as possible. Note that we have to handle an empty line well.

Reply:
Have changed use bufio.Scanner instead of regexp.